docs/utilities.rst

```diff
@@ -27,46 +27,42 @@ The following commands are provided:
 mof_compiler
 ------------
 
-The ``mof_compiler`` command is a command line interface to the pywbem MOF
-compiler. It compiles MOF files, and updates the repository of a WBEM server
-with the result.
+The ``mof_compiler`` command compiles MOF files and updates the CIM repository
+of a WBEM server with the result.
 
-The MOF compiler can also be invoked from programs via the
-:ref:`MOF compiler API`.
+If the compiler fails, any changes made to the CIM repository in the WBEM server
+as part of the current compilation are rolled back.
```

Comment:
Am I missing something? Did we change the qual decls to roll back?

Reply:
Good catch ... we should mention that as a limitation.

Reply:
DONE.
Added: " A limitation is that changes to qualifier declarations are not yet rolled back (see issue #990)."